pkg/controller/services.go

```diff
@@ -1639,21 +1639,30 @@ func (cont *AciController) endpointSliceUpdated(oldobj interface{}, newobj inter
 		cont.log.Error("error processing Endpointslice object: ", newobj)
 		return
 	}
+	proceed := true
+	for _, endpoint := range newendpointslice.Endpoints {
+		if endpoint.Conditions.Ready == nil || !*endpoint.Conditions.Ready {
+			proceed = false
+		}
+	}
+	if !proceed {
+		cont.log.Error("New enpoints are not in ready state")
```

Comment:
should this be an Error log? wouldn't we hit this point even when the pod is in Pending state?